src/plugins/content/accessibility/keyNavigation/strategies/navigatorNavigation.js

```diff
@@ -53,7 +53,47 @@ export default {
         this.managedNavigators = [];
         this.keyNavigators = [];
 
+        let testStatusNavigation;
         if ($navigator.length && !$navigator.hasClass('disabled')) {
+            const $testStatusHeader = $navigator.find('.qti-navigator-info.collapsible .qti-navigator-header');
+            const navigableTestStatus = navigableDomElement.createFromDoms($testStatusHeader);
+
+            if (navigableTestStatus.length) {
+                testStatusNavigation = keyNavigator({
+                    keepState: config.keepState,
+                    id: 'navigator-test-status',
+                    propagateTab: false,
+                    elements: navigableTestStatus,
+                    group: $testStatusHeader,
+                });
+
+                setupItemsNavigator(testStatusNavigation, {
+                    keyNextItem: config.keyNextTab || config.keyNextItem,
+                    keyPrevItem: config.keyPrevTab || config.keyPrevItem
+                });
+                setupClickableNavigator(testStatusNavigation);
```

Comment:
No need to capture the activation of the element. It won't prevent the panel to be collapsed if you remove this line, nor it will add something if you keep it.